static/js/invite.js

```diff
@@ -93,6 +98,19 @@ function submit_invitation_form() {
                 const rendered_email_msg = render_settings_dev_env_email_access();
                 $("#dev_env_msg").html(rendered_email_msg).addClass("alert-info").show();
             }
+
+            if ($("#expires_in").val() === "custom") {
+                // This code hidegit s the custom inputs if value of the custom input was set
```

Comment:
I removed the spurious "git " in this comment, and made a bit more of an edit while I was there.